blocks/header-nav-chain-block/_index.scss

```diff
@@ -0,0 +1,262 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-header-nav-chain {
+	.nav-search {
+		input {
+			@include scss.block-components("header-nav-chain-nav-search-input");
+			@include scss.block-properties("header-nav-chain-nav-search-input");
+		}
+
+		&.open {
+			input {
+				@include scss.block-components("header-nav-chain-nav-search-input-open");
+				@include scss.block-properties("header-nav-chain-nav-search-input-open");
+			}
+
+			button {
+				@include scss.block-components("header-nav-chain-nav-search-button");
+				@include scss.block-properties("header-nav-chain-nav-search-button");
+			}
+		}
+
+		@include scss.block-components("header-nav-chain-nav-search");
+		@include scss.block-properties("header-nav-chain-nav-search");
+	}
+
+	&__flyout-overlay {
+		&::-webkit-scrollbar {
+			@include scss.block-components("header-nav-chain-flyout-overlay-scrollbar");
+			@include scss.block-properties("header-nav-chain-flyout-overlay-scrollbar");
+		}
+
+		&.open {
+			@include scss.block-components("header-nav-chain-flyout-overlay-open");
+			@include scss.block-properties("header-nav-chain-flyout-overlay-open");
+		}
+
+		&.closed {
+			.section-menu {
+				@include scss.block-components("header-nav-chain-flyout-overlay-closed-transition");
```

Comment:
not sure I understand why this token is called "-transition". I know that it's being used to set the value of transition. But it could used for other purposes, right?

Reply:
Fixed